api/v1beta1/openstackdataplanenode_types.go

```diff
@@ -125,7 +125,7 @@ type NetworkConfigSection struct {
 	// +kubebuilder:default=templates/net_config_bridge.j2
 	// Template - ansible j2 nic config template to use when applying node
 	// network configuration
-	Template string `json:"template,omitempty" yaml:"template,omitempty"`
+	Template string `json:"template" yaml:"template"`
```

Comment:
I think we should remove the default, otherwise we may have nodes/roles set with different templates

Reply:
I'm not sure how, but ok. We're not actually using this field for anything yet. We might can remove it in a separate patch.

Reply:
we use when building the inventory: https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/controllers/openstackdataplanenode_controller.go#L277